modules/swagger-codegen/src/main/resources/erlang-server/router.mustache

```diff
@@ -51,6 +62,16 @@ group_paths() ->
         get_operations()
     ).
 
+-spec get_operation({{packageName}}:operation_id()) ->
+   operation_spec().
+
+get_operation(OperationId) ->
```

Comment:
What's not clear for me: who will be responsible for making an instance of an HTTP path when provided with all path parameter values?

Reply:
That would have to be a user code. Do you propose to make it a part of the lib interface?
I.e. a separate api function, which replaces the path Params with provided values?

Reply:
Well, no, not yet.

Reply:
Why not?
In progress )